src/AppDelegate.cpp

```diff
@@ -213,7 +213,7 @@ void AppDelegate::toggleDomainServer(bool start) {
 
 void AppDelegate::toggleAssignmentClientMonitor(bool start) {
     if (start) {
-        _acMonitorProcess->start(QStringList() << "--min" << "4");
+        _acMonitorProcess->start(QStringList() << "--min" << "5");
```

Comment:
Curious why this was a problem, @sethalves correct me if I'm wrong but a --min 5 simply means, keep AT LEAST this number of assignments running at all times, but would also keep a spare AC running which would be grabbed by the remaining assignment on queue and it would then just spawn a new spare?

Given the addition of the new assignment type and the previous information I considered valid, that would mean there was no change required as the --min 4 would simply start with 4 but then spawn a new one to match the number of queued assignments?

Also curious why Brad's list showed all assignments grabbed?